app/models/scoreboard.rb

```diff
@@ -1,7 +1,10 @@
-class ScoreboardSetup < ActiveRecord::Base
```

Comment:
We'll adapt 213 config files to whatever you think best. So don't let 213
stand in the way of improving the scoreboard code.

On Saturday, April 11, 2015, Daniel Bucci <notifications@github.com> wrote:

> In app/models/scoreboard.rb
> <https://github.com/autolab/Autolab/pull/454#discussion_r28196609>:
>
> > @@ -1,7 +1,10 @@
> > -class ScoreboardSetup < ActiveRecord::Base
>
> This is more in line with the Autograder, and it makes more sense from a
> modeling perspective. An Assessment has a Scoreboard or doesn't, and if it
> has a Scoreboard, then that scoreboard's settings will be in the Scoreboard
> model. Plus, this is shorter, which is a plus in my book.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/autolab/Autolab/pull/454/files#r28196609>.
>


-- 
Typed with fat fingers from my iPhone.